seq2science/rules/qc.smk

```diff
@@ -878,15 +878,16 @@ def get_peak_calling_qc(sample):
 
     # deeptools profile
     assembly = treps.loc[sample, "assembly"]
+    narrowpeak_used = get_peak_ftype(list(config["peak_caller"].keys())[0]) == "narrowPeak"
     # TODO: replace with genomepy checkpoint in the future
     if has_annotation(assembly):
         output.extend(expand("{genome_dir}/{{assembly}}/{{assembly}}.annotation.gtf", **config))  # added to be unzipped
         if config.get("deeptools_qc"):
             output.extend(expand("{qc_dir}/plotProfile_gene/{{assembly}}-{peak_caller}.tsv", **config))
-        if get_ftype(list(config["peak_caller"].keys())[0]) == "narrowPeak":
+        if narrowpeak_used:
             output.extend(expand("{qc_dir}/chipseeker/{{assembly}}-{peak_caller}_img1_mqc.png", **config))
             output.extend(expand("{qc_dir}/chipseeker/{{assembly}}-{peak_caller}_img2_mqc.png", **config))
-    if config.get("deeptools_qc"):
+    if config.get("deeptools_qc") and narrowpeak_used:
```

Comment:
can these heatmaps be made if there is only 1 narrowpeak file in the assembly? 

If not, the DESeq2 plotcorrelations in qc.smk should have a similar checking system for the number of treps per assembly.

Reply:
I think so. It clusters on the peaks, not on the pattern. Not 100% sure though